scanomatic/io/scanner_manager.py

```diff
@@ -75,7 +75,9 @@ def _initiate_scanners(self) -> dict[int, ScannerModel]:
         scanner_configs = load(self._paths.config_scanners)
         for scanner in cast(
             list[ScannerModel],
-            [] if scanner_configs is None else scanner_configs,
+            [] if scanner_configs is None else
```

Comment:
two if else in one line feels very iffy.  please at least add parenthesis but preferably rewrite it

Reply:
Refactored this a bit